src/components/flowing-list/flowing-list.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./flowing-list.styles.scss";
+
+export const FlowingList = ({ children }) => {
+  return <div className={"__flowing-list"}>{children}</div>;
```

Comment:
I keep seeing this. Is there a difference between `attribute={"value"}` and `attribute="value"` ?

Reply:
No difference, it's there as this seems to be style used. I don't mind either way